privacy.adoc

```diff
@@ -225,31 +223,31 @@ ____
 [[privacymeasures]]
 === Privacy measures
 
-Several techniques have been developed that helps protect from privacy
+Several techniques have been developed that help protect from privacy
 leaks. Among the most obvious ones are, as
 <<satoshi-unique-addresses,noted by Nakamoto>> above, using unique
 addresses for all transactions, but several others exist. We're not
-going to teach you how to become a privacy ninja, but Bitcoin Q+A has
-a quick summary of privacy enhancing technology, somewhat ordered by
+going to teach you how to become a privacy ninja. However, Bitcoin Q+A has
+a quick summary of privacy-enhancing technology, somewhat ordered by
 how hard they are to implement, at
 https://bitcoiner.guide/privacytips/. You'll notice when you read this
-that Bitcoin privacy often has to do with stuff outside of Bitcoin,
+that Bitcoin privacy often has to do with stuff outside of Bitcoin;
 for example: don't brag about your bitcoins, and use Tor and VPN. The
 post also lists some things directly related to Bitcoin:
 
 Full node:: if you don't use your own full node, you will leak lots of
 information about your wallet to servers on the internet. Running a
 full node is a great first step.
 
-Lightning Network:: Several protocols exists on top of Bitcoin, for
-example the Lightning Network and Blockstream's Liquid sidechain.
+Lightning Network:: Several protocols exist on top of Bitcoin, for
+example, the Lightning Network and Blockstream's Liquid sidechain.
 
 CoinJoin:: a way for multiple people to merge their transactions into
 one, making it harder to do address clustering.
 
 In
 https://btctranscripts.com/breaking-bitcoin/2019/breaking-bitcoin-privacy/[a
-talk] at the Breaking Bitcoin conference Chris Belcher gave an
+talk] at the Breaking Bitcoin conference Chris Belcher, gave an
```

Comment:
Should this comma possibly be moved? 

".. Breaking Bitcoin conference, Chris Belcher gave ..."

Reply:
Never mind! I fixed it in a subsequent commit.